src/workloadaccounts.ts

```diff
@@ -76,11 +74,71 @@ export const WorkloadAccounts = (
   }
 
   return {
-    addAccounts(customer_id: string, spoc_email: string) {
-      for (const orgUnitName of deploymentEnvironments) {
-        addWorkloadAccount(customer_id, spoc_email, orgUnitName)
-      }
+    addAccounts(customer_id: string, spoc_email: string): WorkloadAccount[] {
+      const newWorkloadAccounts = deploymentEnvironments.map(orgUnitName => {
+        return addWorkloadAccount(customer_id, spoc_email, orgUnitName)
+      })
+
       writeAccounts()
+
+      return newWorkloadAccounts
     }
   }
 }
+
+export class WorkloadAccount extends YAMLMap<string, string> {
```

Comment:
This was moved out from the single `types` file, I think I'd prefer them in their own file in a types folder perhaps?